### PR TITLE
General refactoring

### PR DESCRIPTION
**Pull request by necosta**
I performed a general refactoring of the current version just so it looks a bit more presentable.

But my main goal is to start a discussion on what we want to include in this repo in the future.

Adding @mertd and @nunomrc to the discussion

### CODE REVIEW

File: fullstack-challenge/superawesometruck002.json [hunk 1]
```diff
@@ -1,409 +0,0 @@
-[{
```
Comment (nunomrc) [inline fullstack-challenge/superawesometruck002.json]: Not sure if we should delete this file, we need to understand first why is this here in the first place. Maybe this is an example of data that goes with the exercise?

Comment (necosta) [reply fullstack-challenge/superawesometruck002.json]: Yes, @mertd can you give us some more context about this repo? Thanks

Comment (mertd) [reply fullstack-challenge/superawesometruck002.json]: I think Erik was considering publishing some of the recruitment challenges, but I don't know if that is still the current plan.

Comment (necosta) [reply fullstack-challenge/superawesometruck002.json]: Based on my last discussion with him, this repo should contain a mix of easy-to-solve challenges and some more challenging ones.
I suggest we merge and start discussions on Slack about getting more content into this repo.
What do you think?

**Comment by necosta** [comment]
ping @nunomrc , should we merge?
I want to start afterwards a challenge in our team to create challenges to the community 😊

**Comment by nunomrc** [comment]
lets merge :)
